include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -609,7 +614,7 @@ class charge_distribution_surface<Lyt, false> : public Lyt
      *
      * @return Vector of indices describing which SiDBs must be negatively charged.
      */
-    std::vector<int64_t> negative_sidb_detection() noexcept
+    std::vector<int64_t> negative_sidb_detection() const noexcept
```

Comment:
without ``const``

Reply:
I purposefully left this when removing the logically non-const `const` type specifiers. It looks to me that it does not change the state of the `charge_distribution_surface` object or call a function that does. If you intended the function to imply some kind of non-constness I'd be interested to know :)